sky/jobs/state.py

```diff
@@ -232,19 +232,13 @@ async def wrapper(*args, **kwargs):
         for _ in range(_DB_RETRY_TIMES):
             try:
                 return await func(*args, **kwargs)
-            except sqlalchemy_exc.OperationalError as e:
-                last_exc = e
-            except asyncio.exceptions.TimeoutError as e:
-                last_exc = e
-            except OSError as e:
-                last_exc = e
-            except sqlalchemy.exc.TimeoutError as e:
-                last_exc = e
-            except sqlite3.OperationalError as e:
-                last_exc = e
-            except sqlalchemy.exc.InterfaceError as e:
-                last_exc = e
-            except sqlite3.InterfaceError as e:
+            except (sqlalchemy_exc.OperationalError,
+                    asyncio.exceptions.TimeoutError,
+                    OSError,
+                    sqlalchemy.exc.TimeoutError,
```

Comment:
does this exist? I am getting red squiggly. Should be `sqlalchemy_exc`? (underscore)